projWebsite/src/services/productService.ts

```diff
@@ -17,6 +17,17 @@ class ProductService {
         })
     }
 
+    getProduct(id: any){
+        return fetch(MEDEX_API_BASE_URL + 'products/' + id, {
+            method: 'GET',
+            mode: 'cors',
+            headers: {
+                'accept': 'application/json',
+                'Content-Type': 'application/json',
+                'Authorization' : "Bearer "+ TEST_TOKEN
```

Comment:
remove this, use the useAuthStore.getState().token instead

Reply:
Okay, I'll change it.